packages/backend/src/services/UserService.ts

```diff
@@ -356,6 +374,18 @@ export class UserService {
                     loginProvider: 'google',
                 },
             });
+
+            if (
+                Array.isArray(openIdUser.openId.groups) &&
+                openIdUser.openId.groups.length &&
+                sessionUser.organizationUuid
+            )
+                await this.groupsModel.addUserToGroupsIfExist({
+                    userId: sessionUser.userId,
```

Comment:
This can be `sessionUser.userUuid` and the `addUserToGroupsIfExist` method can be cleaner

Reply:
done in: https://github.com/lightdash/lightdash/pull/8508/commits/5ed74ae867c0119c2bf5c9b874aeb204bdc8b252